test/components/urlBarTest.js

```diff
@@ -88,20 +87,5 @@ describe('urlBar', function () {
             .then((val) => val === 'aboutx')
         })
     })
-
-    it('it does not change input after focus until keydown', function * () {
```

Comment:
please change this test to test that focus doesn't bring up any suggestions, thanks